.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+name: ci
```

```suggestion
name: CI
```
Can we have it in capital letters?

---

Yes, I changed. Thank you

Firstly I changed name to 'CI'. But then was needed file renaming. And I tryed lowercase for both. But seems, it is still need uppercase to make GHA badge works.

---

Actually, we don't need the filename to match the name of the workflow. It was perfectly fine with the `ruby.yml` file and `name: CI`.

---

Thank you. I corrected filename.